storage/collections/interfaces/IValidator.go

```diff
@@ -0,0 +1,13 @@
+package interfaces
+
+import (
+	"github.com/bloxapp/ssv/ibft/proto"
+	"github.com/herumi/bls-eth-go-binary/bls"
+)
+
+// IValidator interface for validator storage
+type IValidator interface {
```

Comment:
@nivBlox not sure about separating it to another interface, if it's necessary for the iBFT operation under storage why not put it under the storage interface?

Reply:
also.. if it's a necessary interface it should be in the storage package not implementation

Reply:
@nivBlox @lior-blox not sure about this design..

Reply:
we changed it, now it in the same package.